### PR TITLE
Added vscode-variables to make workspace url variable

### PR DESCRIPTION
**Pull request by DominicVonk**

### CODE REVIEW

**Review by alperg — COMMENTED** [review]
changes look good; could you please answer my questions below?

File: src/extension.ts [hunk 1]
```diff
@@ -274,36 +273,24 @@ class SlackVSCode {
 };
 
 let slack: SlackVSCode;
-
-function cleanupDisposables() {
```
Comment (alperg) [inline src/extension.ts:278]: Why did you remove the cleanupDisposables() ?

File: src/extension.ts [hunk 2]
```diff
@@ -274,36 +273,24 @@ class SlackVSCode {
...
 
   listChannels = CONFIG.get('listChannels');
   listGroups = CONFIG.get('listGroups');
   listMembers = CONFIG.get('listMembers');
   fileWithFullPath = CONFIG.get('fileWithFullPath');
-  excludeFromFullPath = CONFIG.get('excludeFromFullPath');
+  excludeFromFullPath = vscodeVariables(CONFIG.get('excludeFromFullPath'));
   defaultRecipient = CONFIG.get('defaultRecipient');
-
+  console.log('TOKEN', TOKEN);
```
Comment (alperg) [inline src/extension.ts]: Can you remove the console.log?

File: src/extension.ts [hunk 3]
```diff
@@ -274,36 +273,24 @@ class SlackVSCode {
...
+  holder.slack = slack;
   const CONFIG: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('slackVSCode');
   const TOKEN = token = CONFIG.get('token');
 
   listChannels = CONFIG.get('listChannels');
   listGroups = CONFIG.get('listGroups');
   listMembers = CONFIG.get('listMembers');
   fileWithFullPath = CONFIG.get('fileWithFullPath');
-  excludeFromFullPath = CONFIG.get('excludeFromFullPath');
+  excludeFromFullPath = vscodeVariables(CONFIG.get('excludeFromFullPath'));
```
Comment (alperg) [inline src/extension.ts:286]: What vscodeVariables() really do here?

File: src/extension.ts [hunk 4]
```diff
@@ -274,36 +273,24 @@ class SlackVSCode {
...
-
-    try {
-      DISP.dispose();
-    }
-    catch (e) {
-      vscode.window.showErrorMessage('Unknown error.');
-    }
-  }
-}
+let holder = {slack};
```
Comment (alperg) [inline src/extension.ts]: Why do we need `holder` here?

**Comment by DominicVonk** [comment]
Hi which questions?

**Comment by alperg** [comment]
@DominicVonk code review questions that is part of this pull request. click on 'files changed" tab to see them.

**Comment by DominicVonk** [comment]
I think they are hidden by permissions?

**Comment by alperg** [comment]
@DominicVonk can you see them now?

**Comment by DominicVonk** [comment]
@alperg Can you redo the comments? I hope I can see them

File: src/extension.ts (hunk 3 above)
Comment (DominicVonk) [reply src/extension.ts:286]: This is added for the ${workspaceFolder} so you can exclude the workspace folder from the filePath

File: src/extension.ts (hunk 1 above)
Comment (DominicVonk) [reply src/extension.ts:278]: Because it's not doing much

**Comment by DominicVonk** [comment]
Hi see the new commit @alperg

**Review by alperg — APPROVED** [review]

**Comment by alperg** [comment]
@DominicVonk thanks for the PR!
